src/router/index.ts

```diff
@@ -43,4 +43,10 @@ const router = createRouter({
   routes: routes,
 });
 
+router.beforeEach((to) => {
```

Comment:
You shouldn't change the the title of the page until _after_ the the navigation has completed. Doing it this way corrupts the browser's history. Notice, if I navigate from **Home** to **Calculator** to **Top Ten Lists** and then look at my history, all the titles are off:

![image](https://github.com/center-for-threat-informed-defense/top-attack-techniques/assets/79934822/e890cd92-0ed0-4fc3-b275-678c2b695217)

- The Calculator page is called `Top Ten Lists | Top ATT&CK Techniques`
- The Home page is called `Calculator | Top ATT&CK Techniques`

This is because the title of the last page is changed to the title of the new page BEFORE we navigate to the new page, so the last page has its title recorded as the new page in the browser's history.

Currently, the best approach is to perform the work `afterEach` navigation and after all DOM updates are complete using `nextTick`:

```ts
import { nextTick } from "vue";

...

router.afterEach((to) => {
  nextTick(() => {
    document.title = to.meta.title
      ? to.meta.title + " | Top ATT&CK Techniques"
      : "Top ATT&CK Techniques";
  })
})
```
(This code assumes you're storing the page `title` in `meta` as recommended above.)